card/lib/cardio.rb

```diff
@@ -8,7 +8,11 @@
 module Cardio
 
   ActiveSupport.on_load :card do
-    Card::Loader.load_mods if Card.count > 0
+    if Card.count > 0
+      Card::Loader.load_mods 
+    else
+      Rails.logger.warn "empty database"
```

Comment:
I've wasted a lot of debug time because I didn't notice the seeds didn't load and therefor I had no mods loaded ...

I think yes, raise an exception.  Although, I think the point of the test is to make sure it does not throw and exception because lots of mods depend on data (WagnbotID at a minimum).  That would mean we'd have to catch the exception we throw here when we need to load with an empty database.  If not an exception, maybe use 'error' logging level.